streampipes-config/src/main/java/org/apache/streampipes/config/backend/BackendConfig.java

```diff
@@ -128,8 +126,4 @@ private String getJwtSecret() {
   private String makeDefaultJwtSecret() {
     return TokenGenerator.generateNewToken();
   }
-
-  public void updateSetupStatus(boolean status) {
-    config.setBoolean(BackendConfigKeys.IS_SETUP_RUNNING, status);
```

Comment:
`BackendConfigKeys.IS_SETUP_RUNNING` is not used anywhere else, right?
Thus we can remove it